jenkins-pipelines/manager-backup.jenkinsfile

```diff
@@ -10,7 +10,7 @@ longevityPipeline(
     ip_ssh_connections: 'public',
     aws_region: '''["us-east-1", "us-west-2"]''',
     test_name: 'mgmt_cli_test.MgmtCliTest.test_backup_feature',
-    test_config: 'test-cases/manager/manager-regression-multiDC-set-distro.yaml',
+    test_config: 'test-cases/manager/manager-regression-centos-multiDC-set-distro.yaml',
```

Comment:
why did you rename this file ?
the idea was more like:
```
test_config: ['test-cases/manager/manager-regression-multiDC-set-distro.yaml', 'configurations/centos7.yaml']
```
and then the platform specific files are just the parameter needed to be overwritten.

Reply:
I did not know this was possible, I'll change them accordingly